pages/dkp/kommander/2.2/networking/use-istio/index.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: layout.pug
+navigationTitle: Use Istio
+title: Use Istio
+menuWeight: 24
+excerpt: Learn how to integrate microservices managed by Istio into a DKP cluster
+beta: false
+experimental: true
+---
+
+
+Istio helps you manage cloud-based deployments by providing an open-source service mesh to connect, secure, control, and observe microservices.
+
+This topic describes how to expose an application running on the DKP cluster using the LoadBalancer (layer-4) service type.
+
+<p class="message--note"><strong>NOTE: </strong>Usage and installation of Istio on DKP is currently a self-service feature.</p>
+
+## Before you begin
+Before you begin, verify the following:
+
+- You must have access to a Linux, macOS, or Windows computer with a supported operating system version.
+
+- You must have a properly deployed and running cluster.
+
+- You must have `cert-manager` enabled and deployed through your `cluster.yaml` configuration.
+
+## Deploy Istio using DKP
+
+Review the [list of available applications](../../workspaces/applications/platform-applications#workspace-platform-applications) that can be enabled to deploy to your attached cluster.
```

Comment:
Since this page is specifically about enabling the Istio platform-application, do we really need to refer the user to this other page where it is already listed?      I see later in the procedure we need to know the exact version, so perhaps we can improve this like this?
```suggestion
Review the [list of available applications](../../workspaces/applications/platform-applications#workspace-platform-applications) to obtain the current 'APP NAME' for the Istio application, as you will need this name later in this procedure.
```